CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+# Changelog
+
+[npm][1]
+
+[1]: https://www.npmjs.com/package/@google-cloud/vision
+
+## v0.21.0
```

Comment:
How is the version number computed?

Reply:
Releasetool prompts you for it after you write the releasenotes.